src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -1009,15 +1009,7 @@ def aks_create(cmd,     # pylint: disable=too-many-locals,too-many-statements,to
             tier="Paid"
         )
 
-    headers = {}
-    if aks_custom_headers is not None:
```

Comment:
This is not a new parameter? Why there is `aks_custom_headers` before ?

Reply:
@arrownj `aks_custom_headers` is an existing parameter in `aks_create` func, so this is a little refactor in this func, and `aks_custom_headers` is a new parameter in `aks_update` and `aks_agentpool_add` func